lib/fcrepo_wrapper.rb

```diff
@@ -22,7 +22,7 @@ def self.default_instance_options=(options)
   end
 
   def self.default_instance(options = {})
-    @default_instance ||= FcrepoWrapper::Instance.new default_instance_options.merge(options)
```

Comment:
It seems like `default_instance_options` might be entirely unused (and that seems to be the case in `solr_wrapper` too..). Can we remove that entirely?